lib/spack/spack/stage.py

```diff
@@ -657,17 +642,18 @@ def cache_local(self):
         tty.msg("Sources for DIY stages are not cached")
 
 
-def ensure_access(file=spack.paths.stage_path):
+def ensure_access(file_):
```

Comment:
what is this underscore?

Reply:
Checkers like pylint will tell you you're redefining a built-in and appending an underscore is a simple way to avoid the issue.

Reply:
Removing